packages/gatsby-plugin-coffeescript/src/__tests__/gatsby-node.js

```diff
@@ -46,7 +62,7 @@ describe(`gatsby-plugin-coffeescript`, () => {
           },
           {}
         )
-      ).toMatchSnapshot()
+      ).toEqual(`complied-code`)
```

Comment:
can we actually not mock it and use `toMatchInlineSnapshot` so it ends up in same file (and looks pretty much the same as `toEqual` assertion? I feel like mocking too much (unnecessarily) can hide some actual problems